dsgrid/notebooks/start_notebook.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+unset XDG_RUNTIME_DIR
+export SPARK_CLUSTER=$1
+export DSGRID_LOG_FILE_PATH=`pwd`/$2
+echo "Spark cluster is running at ${SPARK_CLUSTER}" >&2
+echo "JADE output directory is ${DSGRID_LOG_FILE_PATH}" >&2
+mkdir -p $DSGRID_LOG_FILE_PATH
+jupyter notebook --no-browser --ip=0.0.0.0 --port 8889 &
+sleep 10
+echo "Create an ssh tunnel with this command: ssh -L 8889:${HOSTNAME}:8889 ${USER}@el1.hpc.nrel.gov" >&2
+wait
```

Comment:
Drawbacks here compared to https://github.com/dsgrid/dsgrid-coordination/blob/main/prep_spark_cluster_notebook.py (and instructions on that README):
- Not obvious how to start and restart the jupyter notebook if that becomes necessary/desired
- Jade job won't finish elegantly and so won't copy performance stats back to the run folder
- No links to Spark UI (although I still don't know how to use that anyway)

Reply:
Good point about a clean shutdown. I'll add instructions. The Spark UI discussion is another case where we need one main site to link against.

Reply:
Please refer to the README instructions that I added.